### PR TITLE
Update to rails 8

### PR DESCRIPTION
**Pull request by chvp**

### CODE REVIEW

**Comment by chvp** [comment]
`active_model_serializers` doesn't seem to be compatible with Rails 8. It's not maintained at all, so I'll switch away from it. (But I'll do that in a separate MR.)

**Comment by chvp** [comment]
Based on #681, that should be looked at and merged first.

File: config/puma.rb [hunk 1]
```diff
@@ -19,16 +27,12 @@
...
-state_path ENV.fetch('STATEPATH') { 'tmp/pids/server.state' }
-bind ENV.fetch('SOCKETFILE') { 'unix://tmp/sockets/puma.sock' }
 
-# Allow puma to be restarted by `rails restart` command.
+# Allow puma to be restarted by `bin/rails restart` command.
 plugin :tmp_restart
+
+# Specify the PID file. Defaults to tmp/pids/server.pid in development.
+# In other environments, only set the PID file if requested.
+pidfile ENV['PIDFILE'] if ENV['PIDFILE']
```
Comment (robbevp) [inline config/puma.rb:38]: We lost the config for state_path and socketfile config here, but I think we want to keep these?

**Review by robbevp — CHANGES_REQUESTED** [review]

File: config/puma.rb (hunk 1 above)
Comment (chvp) [reply config/puma.rb]: Good catch, fixed.

**Review by robbevp — APPROVED** [review]

**Pull request merged**